app/controllers/api/v0/markets_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V0::MarketsController < ApplicationController
+    def index
+        render json: Market.count_vendors
+    end
+
+    def update
+
+    end
```

Comment:
Whoops, I had you add this in and we forgot to take it out! We can remember to remove it later